lib/portus/ldap.rb

```diff
@@ -54,6 +54,35 @@ def self.enabled?
 
     protected
 
+    # Returns auth options according to configuration.
+    def auth_options
+      cfg = APP_CONFIG["ldap"]
+      {
+        auth: {
+          username: cfg["authentication"]["bind_dn"],
+          password: cfg["authentication"]["password"],
+          method: :simple
+        }
```

Comment:
What rubocop wants is the following:

```ruby
auth: {
  username: cfg["authentication"]["bind_dn"],
  password: cfg["authentication"]["password"],
  method:   :simple
}
```

That is, put spaces before `:simple` so it's aligned to the other elements.